Initium-Odp/war/odp/twoDimSandbox.jsp

```diff
@@ -0,0 +1,50 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" import="com.universeprojects.miniup.server.RandomTileGenerator" %>
+
+<html>
+<head>
+    <title>Sandbox 2D Tiles</title>
+</head>
+<body id="somebody">
+
+Width: <input type='text' id='width' value=20 />
+Height: <input type='text' id='height' value=20 />
+Seed: <input type='text' id='seed' value=123456 />
+<button id="somebutton">press here</button>
+
+<script src="http://code.jquery.com/jquery-latest.min.js"></script>
+<script>
+    $(document).on("click", "#somebutton", function() {
+        $.ajax({
+            url: "SandboxServlet",
+            data:{width:$("#width").val(), height:$("#height").val(), seed:$("#seed").val()},
+            type: 'POST',
+            success: function(responseJson) {
+
+                // Remove all current images
+                var images = document.getElementsByTagName('img');
```

Comment:
This map will be displayed a
Inside the current game page in place of the banner. This JavaScript would make all images disappear, not just the ones in the map I think. You might also consider doing this the jQuery way since it's super nice for this kind of stuff.